_includes/docs-sidebar.html

```diff
@@ -0,0 +1,27 @@
+<div class = "sidebar">
+  <div class="sidebar-content">
+   <!-- Add new sections here -->
+   {% assign order = 
+      "Dev Setup,
+      Add to Team Page"
+    %}
```

Comment:
You could pull this information directly from `{{ site.posts }}`.

Reply:
I think `{{site.docs}}` instead - the `posts` collection is something else!